packages/core/src/generators/react.ts

```diff
@@ -160,7 +163,9 @@ export const blockToReact = (json: JSXLiteNode, options: ToReactOptions) => {
         str += ` ${BINDING_MAPPERS[key]}={${useBindingValue}} `;
       }
     } else {
-      str += ` ${key}={${useBindingValue}} `;
+      if (!isValidAttributeName(key)) {
```

Comment:
```suggestion
      if (isValidAttributeName(key)) {
```

this should fix the snapshots

Reply:
fixed